plutus-ledger-constraints/src/Ledger/Constraints/OnChain/V1.hs

```diff
@@ -118,7 +118,7 @@ checkTxConstraint ctx@ScriptContext{scriptContextTxInfo} = \case
             addr = Address.scriptHashAddress vlh
             checkOutput TxOut{txOutAddress, txOutValue, txOutDatumHash=Just svh} =
                    Ada.fromValue txOutValue >= Ada.fromValue vl
-                && Ada.fromValue txOutValue <= Ada.fromValue vl + Ledger.maxMinAdaTxOut
```

Comment:
One more here.

Reply:
Fixed by "Correct to use maxMinAdaTxOut. 2".